### PR TITLE
Alan/rs 5 top menu navigation on landing pages

### PR DESCRIPTION
**Pull request by alanbiju79**
image

### CODE REVIEW

**Comment by danielpaul** [comment]
@alanbiju79 the devise pages are all top aligned now. they should still be centre aligned. also the flash messages are coming under the nav bar. the z-index needs to be adjusted so it appears above it. 

CleanShot 2023-05-30 at 18 53 03@2x
CleanShot 2023-05-30 at 18 53 31@2x

File: app/controllers/static_controller.rb
Comment (danielpaul) [inline app/controllers/static_controller.rb:1]: there's this file that can also be removed? `spec/requests/static_spec.rb`

**Review by danielpaul — CHANGES_REQUESTED** [review]

File: app/controllers/static_controller.rb
Comment (danielpaul) [reply app/controllers/static_controller.rb:1]: and there is also a line in the routes

Comment (alanbiju79) [reply app/controllers/static_controller.rb:1]: removed in the next PR

**Comment by alanbiju79** [comment]
> @alanbiju79 the devise pages are all top aligned now. they should still be centre aligned. also the flash messages are coming under the nav bar. the z-index needs to be adjusted so it appears above it.
> 
> CleanShot 2023-05-30 at 18 53 03@2x CleanShot 2023-05-30 at 18 53 31@2x

the z-index is fixed in the next pr.
image
The devise pages are centre aligned for me. Also I didn't touch the devise layout.

**Comment by danielpaul** [comment]
@alanbiju79 could you move the changes that we need to this PR? we can only merge this PR once everything is there.

**Comment by danielpaul** [comment]
There is this middle view that is a bit strange. we only need the mobile layout menu all the down or maybe we hide the logo so there isn't maybe two logos at this size maybe?

image

**Comment by danielpaul** [comment]
Around the devise pages it's a bit hard to see on smaller screen. try the forgot password or confirmation instructions page to replicate it?

image

File: app/views/shared/_navbars.html.haml [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+            .absolute.right-0.z-10.w-32.origin-top-right.rounded-md.bg-white.dark:bg-card-dark.py-2.shadow-lg.ring-1.focus:outline-none.text-gray-900.dark:text-white{"aria-labelledby" => "user-menu-button", "aria-orientation" => "vertical", :class => "mt-2.5 ring-gray-900/5", :role => "menu", :tabindex => "-1", "x-show": "dropdownOpen", "@click.outside": "dropdownOpen = false", "x-transition:enter": "ease-out duration-100", "x-transition:enter-start": "transform opacity-0 scale-95", "x-transition:enter-end": "transform opacity-100 scale-100", "x-transition:leave": "transition ease-in duration-75", "x-transition:leave-start": "transform opacity-100 scale-100", "x-transition:leave-end": "transform opacity-0 scale-95"}
+              -# %a#user-menu-item-0.block.px-3.py-1.text-sm.leading-6.text-gray-900{:href => "#", :role => "menuitem", :tabindex => "-1"} Your profile
+              = link_to "Profile", edit_user_registration_path, class: "block px-3 py-1 text-sm leading-6 hover:bg-gray-50 dark:hover:bg-body-dark"
+              = link_to "Sign Out", destroy_user_session_path, method: :delete, class: "block px-3 py-1 text-sm leading-6 hover:bg-gray-50 dark:hover:bg-body-dark"
```
Comment (alanbiju79) [inline app/views/shared/_navbars.html.haml:109]: @danielpaul can you have a look at this sign_out, when I click it I get "No route matches [GET] '/users/sign_out'". I checked the routes and the destroy_user_session path is there

File: app/views/layouts/application.html.haml [hunk 2]
```diff
@@ -1,9 +1,9 @@
 !!!
-%html{lang: "en"}
+%html.h-full{lang: "en"}
   %head
     = render "layouts/components/head"
 
   %body
-    = yield
+    = render "shared/navbars"
```
Comment (danielpaul) [inline app/views/layouts/application.html.haml:7]: we should still have a yield? that's where the content will get rendered no?

File: app/views/shared/_navbars.html.haml
Comment (danielpaul) [inline app/views/shared/_navbars.html.haml:1]: this is missing the notifications icon 

CleanShot 2023-05-31 at 16 17 18@2x

File: spec/requests/dashboard_spec.rb
Comment (danielpaul) [inline spec/requests/dashboard_spec.rb:1]: we can delete this file

**Review by danielpaul — CHANGES_REQUESTED** [review]

File: app/views/layouts/application.html.haml (hunk 2 above)
Comment (alanbiju79) [reply app/views/layouts/application.html.haml:7]: so the yield is in the navbars partial. It's done this way so the content renders in the correct place

**Pull request merged**